src/lightning/pytorch/loggers/logger.py

```diff
@@ -120,25 +120,50 @@ def merge_dicts(  # pragma: no cover
         >>> d1 = {'a': 1.7, 'b': 2.0, 'c': 1, 'd': {'d1': 1, 'd3': 3}}
         >>> d2 = {'a': 1.1, 'b': 2.2, 'v': 1, 'd': {'d1': 2, 'd2': 3}}
         >>> d3 = {'a': 1.1, 'v': 2.3, 'd': {'d3': 3, 'd4': {'d5': 1}}}
-        >>> dflt_func = min
-        >>> agg_funcs = {'a': np.mean, 'v': max, 'd': {'d1': sum}}
+        >>> dflt_func = torch.min
+        >>> agg_funcs = {'a': torch.mean, 'v': torch.max, 'd': {'d1': torch.sum}}
         >>> pprint.pprint(merge_dicts([d1, d2, d3], agg_funcs, dflt_func))
-        {'a': 1.3,
-         'b': 2.0,
-         'c': 1,
-         'd': {'d1': 3, 'd2': 3, 'd3': 3, 'd4': {'d5': 1}},
-         'v': 2.3}
+        {'a': tensor(1.3000),
```

Comment:
```
{'a': 1.3,
         'b': 2.0,
         'c': 1,
         'd': {'d1': 3, 'd2': 3, 'd3': 3, 'd4': {'d5': 1}},
         'v': 2.3}
```

these variables were Python builtin, not sure if we want to replace that with torch.Tensor. @Borda ?